helevents/templates/rest_framework/event_list.html

```diff
@@ -366,6 +366,14 @@ <h4 id="hide-recurring-children">Hide recurring children</h4>
         <pre><code>event/?hide_recurring_children=true</code></pre>
         <p><a href="?hide_recurring_children=true" title="json">See the result</a></p>
 
+        <h4 id="sub-events">Sub events</h4>
+        <p>You may use the query parameter <code>sub_events</code> in pair with <code>hide_recurring_children</code> to apply other filters to child events.</p>
```

Comment:
It would probably be a good idea to state that it will not affect all filters (like the ones processed in `_filter_event_queryset`)?

Reply:
Now i changed this to be applied with `_filter_event_queryset ` filters as well.